requirements.txt

```diff
@@ -1,4 +1,8 @@
+deepdiff
+GitPython
```

Comment:
If I recall correctly, `GitPython` is just a wrapper around `git`. I think we should mention `git` as a requirement as well. I know `git` si needed to clone the repo, but in case this evolves and at some point we have a Docker image, we cannot just simply copy the files into the image and install python and go.